File: App/Backend/Backend/Backend/Models/UploadUserImage.cs
using Backend.Utils;
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Backend.Models
{
    public class UploadUserImage
    {
        public string Username { get; set; }
        public string ImageName { get; set; }
        public string OldImageName { get; set; }
        [JsonConverter(typeof(Base64FileJsonConverter))]
        public byte[] ImageData { get; set; }
    }
}

File: App/Frontend/frontendWeb/lib/models/uploadUserImage.dart
class UploadUserImage {
  String username;
  String imageData;

  UploadUserImage(this.username, this.imageData);

  Map<String, dynamic> toJSON() {
    var map = Map<String, dynamic>();

    map["username"] = this.username;
    map["imageData"] = this.imageData;

    return map;
  }
}